legal-api/tests/unit/services/filings/validations/test_alteration.py

```diff
@@ -244,7 +302,7 @@ def test_validate_numbered_name(session, test_name, legal_type, new_legal_type,
         # check that validation failed
         assert err
         assert HTTPStatus.BAD_REQUEST == err.code
-        assert err.msg[0]['error'] == err_msg
+        assert err_msg in err.msg[0]['error']
```

Comment:
I changed this so we don't have to use the full, 2-line, error messages in the test parameters.